src/apps/btc/btc_sign.c

```diff
@@ -580,7 +584,7 @@ app_btc_sign_error_t app_btc_sign_output(
             return _error(APP_BTC_SIGN_ERR_USER_ABORT);
         }
 
-        sha256_finish(&_hash_outputs_ctx, _hash_outputs);
+        rust_sha256_finish(&_hash_outputs_ctx, _hash_outputs);
         // hash hash_outputs to produce the final double-hash
         _sha256(_hash_outputs, 32, _hash_outputs);
```

Comment:
for instance here, would `rust_sha256(_hash_outputs, 32, _hash_outputs);` work as well?